go.mod

```diff
@@ -96,12 +97,12 @@ require (
 	github.com/russellhaering/goxmldsig v1.4.0
 	github.com/segmentio/analytics-go/v3 v3.2.1
 	github.com/sergi/go-diff v1.3.1
-	github.com/sigstore/cosign/v2 v2.0.2
+	github.com/sigstore/cosign/v2 v2.2.2
```

Comment:
Why it was changed? We have a replace for cosign. Do we need to rebase our fork? https://github.com/stackrox/stackrox/blob/5f3cfdea3d40b8a09cca0a9bc968711bd2041e00/go.mod#L460
cc @dhaus67

Reply:
Don't know why, but also `master` and `release-4.4` use `github.com/sigstore/cosign/v2 v2.2.2`